### PR TITLE
Feature/add dist build

### PR DESCRIPTION
**Pull request by LucaMele**
Adds a IIFE bundle parallel to every lib folder. Only for legacy. No common chunks

https://jsbin.com/ruxubequma/2/edit?html,js,output

### CODE REVIEW

**Comment by jackblackCH** [comment]
Might be offtopic but npm scripts look a bit polluted with rimraf and cleanup scripts. is it possible to put them away from the "public" api

**Comment by LucaMele** [comment]
agree with marc

File: rollup.config.js [hunk 1]
```diff
@@ -20,45 +22,102 @@ const babelOptions = require('./.storybook/.babelrc'); // get the babelrc file
...
+    babel({
+      presets: babelOptions.presets,
+      plugins: [
+        ...babelOptions.plugins,
+        '@babel/plugin-transform-runtime'
+      ],
+      babelrc: false,
+      exclude: [
+        'node_modules/@skatejs/val/**',
+        'node_modules/@babel/**',
```
Comment (codepleb) [inline rollup.config.js]: Why are only those two node_modules folders ignored?

Comment (LucaMele) [reply rollup.config.js]: @codepleb  Cause they dont provide correctly UMD named exports and generate a warning in rollup build. The warning says that those modules are ignored and threated as external. This settings is only here to suppress that warning

**Comment by raphaellueckl** [comment]
lgtm

**Pull request merged**